test/helpers/mocks.dart

```diff
@@ -79,3 +79,21 @@ class MockAccountBalanceNotifier
   @override
   FutureOr<AccountBalance?> build() async => accountBalance;
 }
+
+sealed class MockTransactionNotifierType {
+  const MockTransactionNotifierType();
+}
+
+class MockTransactionNotifierWithData extends MockTransactionNotifierType {
+  const MockTransactionNotifierWithData({required this.transactionType});
+
+  final TransactionType transactionType;
+}
+
+class MockTransactionNotifierWithNullData extends MockTransactionNotifierType {
+  const MockTransactionNotifierWithNullData();
+}
+
+class MockTransactionNotifierWithError extends MockTransactionNotifierType {
+  const MockTransactionNotifierWithError();
+}
```

Comment:
These lines are still duplicated in `transaction_tile_test.dart`. 

https://github.com/TBD54566975/didpay/blob/0bc1feb11fc2f4a00f0527c81381cb36436de3df/test/features/transaction/transaction_tile_test.dart#L186-L202

They can be removed from the `transaction_tile_test.dart` so that it only uses the ones from `mocks.dart`.

Reply:
So, should I remove these lines from transaction_tile_test.dart also?
@victoreronmosele